io.github.santiagocezar.maniatic-launcher.yaml

```diff
@@ -0,0 +1,68 @@
+
+app-id: io.github.santiagocezar.maniatic-launcher
+runtime: org.gnome.Platform
+runtime-version: "42"
+sdk: org.gnome.Sdk
+command: maniatic
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --persist=.
+  # gamepads
+  - --device=all
+  # to copy the Data.rsdk from user installed Sonic Mania
+  - --filesystem=~/.steam:ro
+  - --filesystem=~/.var/app/com.valvesoftware.Steam:ro
+modules:
+  - name: glfw
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=RelWithDebInfo
+      - -DBUILD_SHARED_LIBS:BOOL=ON
+      - -DBUILD_GLFW_EXAMPLES:BOOL=OFF
+      - -DBUILD_GLFW_DOCS:BOOL=OFF
+    sources:
+      - type: archive
+        url: https://github.com/glfw/glfw/releases/download/3.3.8/glfw-3.3.8.zip
+        sha256: 4d025083cc4a3dd1f91ab9b9ba4f5807193823e565a5bcf4be202669d9911ea6
+    cleanup:
+      - /include
+      - /lib/cmake
+      - /lib/pkgconfig
+  - shared-modules/glew/glew.json
+  - shared-modules/glu/glu-9.json
+  - name: rsdkv5
+    no-autogen: true
+    no-make-install: true
+    make-args:
+      - RSDK_ONLY=1
+      - RSDK_REVISION=3
+      # some default flags were causing problem at runtime
+      - CXXFLAGS=
```

Comment:
which one? 

they have a definitive purpose.

Reply:
Ok, bisected the flags and it was `-Wp,-D_FORTIFY_SOURCE=2` what was causing problems. Makes sense as the logging and .ini writing functions were being affected. Will report to upstream